### PR TITLE
added Quicken Loans

### PR DESCRIPTION
**Pull request by waterskier2007**
Adds or updates the following companies, events, or universities:
 - Quicken Loans

#### Company updates
 - [x] I have put the most recent relevant date in the "Last Update" column
 - [ ] I have linked to the article about the change, not the company's homepage (Please put N/A if there is no public post) - N/A

#### Event updates
 - [ ] I have linked to the article about the change, not the event's homepage

#### University updates
 - [ ] I have linked to the article about the change, not the university's homepage

### CODE REVIEW

File: _data/companies.yml [hunk 1]
```diff
@@ -2734,3 +2734,10 @@
   visitors: Restricted
   events: Restricted
   last_update: 2020-03-13
+ 
+ - name: Quicken Loans
+  wfh: Encouraged
+  travel: Restricted
+  visitors: "?"
```
Comment (QualityRobD) [inline _data/companies.yml]: @waterskier2007, `vistors` should be listed as `Restricted` per the company email we received yesterday.

Comment (waterskier2007) [reply _data/companies.yml]: updated

Comment (QualityRobD) [reply _data/companies.yml]: 🚀

**Comment by phildini** [comment]
Closing due to rebase and staleness. Feel free to re-open after fixes, thanks for your patience.

**Pull request closed without merging**